source/Mlos.Python/mlos/Spaces/HypergridAdapters/CategoricalToOneHotEncodedHypergridAdapter.py

```diff
@@ -0,0 +1,268 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+from typing import Dict
+import numpy as np
+from pandas import DataFrame
+from sklearn.preprocessing import OneHotEncoder
+from mlos.Spaces import CategoricalDimension, DiscreteDimension, Hypergrid, SimpleHypergrid
+from mlos.Spaces.HypergridAdapters.HypergridAdapter import HypergridAdapter
+from mlos.Spaces.HypergridAdapters.HierarchicalToFlatHypergridAdapter import HierarchicalToFlatHypergridAdapter
+from mlos.Spaces.HypergridAdapters.CategoricalToDiscreteHypergridAdapter import CategoricalToDiscreteHypergridAdapter
+
+
+class CategoricalToOneHotEncodingAdapteeTargetMapping:
+    """ Retains the list of target Hypergrid's (one hot encoded) dimensions

```

Comment:
@amueller to comment more, but I think these are supposed to not have the space separation in order to be processed by `sphinx` for API publication.